### DISCUSSION TITLE
Is my .gitattributes correct?

### DISCUSSION THREAD

**Discussion by Ismoh** [github-linguist/linguist#5704] [Q&A] [answered]
Am I impatient or is there anything wrong in my .gitattributes?
```
# Try to exclude everything from linguist
.building/* linguist-vendored
.debug/* linguist-vendored
.vscode/* linguist-vendored

# Include only self implemented code
.github/workflows/* linguist-detectable
.testing/tests/* linguist-detectable
noita-mp/* linguist-detectable
```
https://github.com/Ismoh/NoitaMP/blob/master/.gitattributes

My expectation is that there only be Lua and YAML shown, but nothing changed since days.
![image](https://user-images.githubusercontent.com/12631485/147293984-b7e70dbb-c598-4757-b54b-cf4d66deba8e.png)

Looks like I am too stupid to install linguist locally on windows..

**Comment by Ismoh** [comment, marked as answer]
Fixed it!
https://github.com/Ismoh/NoitaMP/blob/develop/.gitattributes

**Reply by lildude** [reply]
Yes, this is expected as the overrides for language detection in the sidebar etc only affect the primary branch of a repo, which in your case is the `develop` branch.

**Reply by Ismoh** [reply]
Ah, I wasnt aware of!
If default is master, then masters gitattribute, I guess?! 
Thank you and enjoy your day!
